docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -59,4 +60,84 @@ let lockScreenConfiguration = SDLLockScreenConfiguration.enabledConfiguration(wi
 ```
 
 ### Using the Vehicle's Icon
-If you want to build your own lock screen view controller, it is recommended that you subclass `SDLLockScreenViewController` and use the public `appIcon`, `vehicleIcon`, and `backgroundColor` properties.
\ No newline at end of file
+If you want to build your own lock screen view controller, it is recommended that you subclass `SDLLockScreenViewController` and use the public `appIcon`, `vehicleIcon`, and `backgroundColor` properties.
+!@
+
+@![android, javaSE, javaEE] `// TODO, Android and iOS guides on this section are very different should they be aligned better?`
```

Comment:
The Android and iOS sections share the many of the same headers. Try to reuse the headers and fit the different content under them.

Reply:
* Lock Screen Activity - Android only
* Using the Provided Lock Screen
    * Android content is under their "Configurations" header. Lots of duplicate stuff with our header so use your best judgement when leaving/removing stuff. 
* Customizing the Provided Lock Screen
    * Custom BG Color 
        * modify the iOS code to fit here
    * Custom App Icon
        * modify the iOS code to fit here
    * Showing the Vehicle's Icon
        * modify the iOS code to fit here
* Using Your Own Lock Screen 
   * This has *Setting a Custom Lock Screen View* content from Android
* Disabling the lock screen manager
    * iOS needs to implement this section

Reply:
updated